src/test/groovy/org/prebid/server/functional/service/PrebidServerService.groovy

```diff
@@ -287,6 +287,10 @@ class PrebidServerService implements ObjectMapperWrapper {
         response.body().asString()
     }
 
+    void withWarmup(){
```

Comment:
I proposed to return `this` so that you can call it fluently without adding an additional line of code each time you wanted to do a warmup.